src/Health-monitor.js

```diff
@@ -14,9 +14,11 @@ class HealthMonitor {
 
   calculateDaily(id, date, property) {
     const propertyByDate = this.dataSet.find(data => {
-      return id === data.userID && date == data.date
+      return id === data.userID && date === data.date
     });
-    return propertyByDate[property];
+    if (propertyByDate && propertyByDate[property]) {
```

Comment:
We should look at this more closely to see what NaN is popping up in those fields on our page. This may have something to do with it.